lib/tasks/data_feeds/dark_sky_temperature_loader.rake

```diff
@@ -0,0 +1,37 @@
+namespace :data_feeds do
+  def process_area(area, start_date, end_date)
```

Comment:
Possibly a bit much going on here for a rake task? It falls outside the bounds of our usual testing areas.

Reply:
Yep, fair enough